src/Uno.UWP/ApplicationModel/Chat/ChatMessageManager.Android.cs

```diff
@@ -0,0 +1,57 @@
+﻿#if __ANDROID__
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Android.Content;
+using Android.Net;
+using Android.Provider;
+using Uno.UI;
+using Windows.Foundation;
+
+namespace Windows.ApplicationModel.Chat
+{
+	public partial class ChatMessageManager
+	{
+		private const string SmsProtocol = "smsto:";
+		private const string SmsBodyExtra = "sms_body";
+		private const string RecipientsSeparator = ";";
+
+		public static IAsyncAction ShowComposeSmsMessageAsync(ChatMessage message)
+		{
+			if (message == null)
+			{
+				throw new ArgumentNullException(nameof(message));
+			}
+
+			var intent = CreateIntent(message.Body, message.Recipients.ToArray())
+				.SetFlags(ActivityFlags.ClearTop)
+				.SetFlags(ActivityFlags.NewTask);
+
+			ContextHelper.Current.StartActivity(intent);
+			
+			return Task.FromResult(true).AsAsyncAction();
```

Comment:
On iOS the task completes once the message has been sent, on Android it completes synchronously as soon as the intent is launched... which is the correct behaviour? ('Correct' == matching UWP)

Reply:
That is a good point. I will try to dig up my old Lumia to check. On desktop this just opens the deprecated SMS app, so it is not very useful and can't really be used to check.